source/tests/system/nixnetsocket_driver_api_tests.cpp

```diff
@@ -136,6 +136,27 @@ TEST_F(NiXnetDriverApiTests, InvalidSocket_Select_ReturnsAndSetsExpectedErrors)
   EXPECT_XNET_ERROR(SOCKET_COULD_NOT_BE_FOUND_ERROR, select_last_error);
   EXPECT_THAT(SOCKET_COULD_NOT_BE_FOUND_MESSAGE, select_last_error_str.error());
 }
+
+TEST_F(NiXnetDriverApiTests, Socket_GetSocketOptionMulticastTTL_ResponseIsReasonable)
```

Comment:
These tests fail on EXPECT_SUCCESS with -1 response. Not sure if we're at the point to be able to test these or not, need to work on them and cross reference with SocketTests.cpp.